news/_posts/2015-05-05-release-notes-2.12.0-M1.md

```diff
@@ -0,0 +1,163 @@
+Jason Zaugg, Lukas Rytz, A. P. Marki, Rex Kerr, Kato Kazuyoshi, Max Bileschi, jxcoder, François Garillot, rubyu, Adriaan Moors, Dominik Gruntz, Evgeny Vereshchagin, Marc Siegel, Masato Sogame, Simon Ochsenreither, Todd Vierling, Viktor Klang, cchantep, Denton Cockburn, Paolo Giarrusso, Denis Rosset, Roman Hargrave, Rui Gonçalves, Shadaj, harryhuk, Steven Scott, Antoine Gourlay, Aleksandar Prokopec, Lukas Elmer, Erlend Hamnaberg, Maks Atygaev, Malte Isberner, dgruntz.
+Thank you!
+
+These release notes are [hosted on GitHub](https://github.com/scala/make-release-notes/blob/2.12.x/hand-written.md) and are continuously updated during the Scala 2.12.0 release cycle.
+You are kindly invited to contribute!
+
+
+### Reporting Bugs
+
+Please file any bugs you encounter on [our issue tracker](https://issues.scala-lang.org).
+If you're unsure whether something is a bug, please contact the [scala-user](https://groups.google.com/forum/?fromgroups#!forum/scala-user) mailing list.
+Before creating a new issue, search search the issue tracker to see if your bug has already been reported.
+
+
+### Scala IDE for Eclipse
+
+A release of the Scala IDE for Eclipse for Scala 2.12 will be available together with the release.
+
+Note that for the current milestone (2.12.0-M1), the Scala IDE is not yet available.
+
+<!-- Notes from 2.11.0
+The Scala IDE with this release built in is [available from this update site](http://download.scala-ide.org/sdk/helium/e38/scala211/stable/site/) for [Eclipse 4.2/4.3 (Juno/Kepler)](http://www.eclipse.org/downloads/packages/eclipse-ide-java-developers/keplersr2). Please have a look at the [getting started guide](http://scala-ide.org/docs/user/gettingstarted.html) for more info.
+-->
+
+
+### Available Projects
+
+Please refer to the list of [libraries and frameworks available for Scala 2.12](https://github.com/scala/make-release-notes/blob/2.12.x/projects-2.12.md).
+
+
+### Binary Compatibility
+
+When two versions of Scala are binary compatible, it is safe to compile your project on one Scala version and link against another Scala version at run time.
+Safe run-time linkage (only!) means that the JVM does not throw a (subclass of) [`LinkageError`](http://docs.oracle.com/javase/7/docs/api/java/lang/LinkageError.html) when executing your program in the mixed scenario, assuming that none arise when compiling and running on the same version of Scala.
+Concretely, this means you may have external dependencies on your run-time classpath that use a different version of Scala than the one you're compiling with, as long as they're binary compatibile.
+In other words, separate compilation on different binary compatible versions does not introduce problems compared to compiling and running everything on the same version of Scala.
+
+We check binary compatibility automatically with [MiMa](https://github.com/typesafehub/migration-manager).
```

Comment:
Maybe for next release we should refactor the bulk of the BC verbiage to a dedicated page.